Cake.Recipe/Content/build.cake

```diff
@@ -330,9 +330,9 @@ public void CopyBuildOutput(BuildVersion buildVersion)
             continue;
         }
 
-        if (parsedProject.OutputPath == null || parsedProject.RootNameSpace == null || parsedProject.OutputType == null)
+        if (parsedProject.OutputPaths[0] == null || parsedProject.RootNameSpace == null || parsedProject.OutputType == null)
```

Comment:
Is there a guarantee that OutputPaths itself won't be null, and it will contain at least 1 item?

Reply:
Honestly, I have no idea :-)  I don't think I have a non dotnetcorebuild project that uses Cake.Recipe anymore, so not really in a position to test this.

Reply:
Me neither, I archived the last one a few years ago.
Well, let us cross that bridge when that time comes.